x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.tsx

```diff
@@ -84,6 +84,9 @@ export const CspNoDataPage = ({
           display: block;
           margin: auto;
           width: 450px;
+          div[class*='euiCard__footer'] {
+            display: contents;
+          }
```

Comment:
what exactly does this fix? can you provide before and after pics?
[display: contents](https://caniuse.com/css-display-contents) seems pretty buggy causing buttons and HTML elements not to work when displayed, combined with placing directly on a "private" eui class can be a bit alarming.

Reply:
<img width="767" alt="Screen Shot 2023-02-02 at 7 26 10 PM" src="https://user-images.githubusercontent.com/8703149/216505738-70b82ccb-934f-49de-91df-f89b5cc9ba23.png">
<img width="735" alt="Screen Shot 2023-02-02 at 7 27 12 PM" src="https://user-images.githubusercontent.com/8703149/216505741-2623754a-875b-4630-ab13-fe7b447eeed7.png">

It shifts the button for the onboarding a bit to the center, you can see the comparison without using it (top) and after using it (bottom)

Reply:
I'm still a bit hesitant about this fix, as I mentioned, display content seems to have bugs, check the can-i-use page that I posted to read more about them. Also, I would rather if we avoid using classes that were not exposed to changes.